<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SumanSuma" ignored="0" name="Default test" tests="1" failures="0" timestamp="2023-08-01T20:56:49 IST" time="0.066" errors="0">
  <testcase name="@BeforeClass bcConfig" time="0.0" classname="Vtiger.GenericUtilities.BaseClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Vtiger.GenericUtilities.BaseClass.bcConfig(BaseClass.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass bcConfig -->
  <testcase name="@BeforeMethod bmConfig" time="-1.690903609906E9" classname="Vtiger.GenericUtilities.BaseClass">
    <skipped/>
  </testcase> <!-- @BeforeMethod bmConfig -->
  <testcase name="@AfterMethod amConfig" time="-1.690903609922E9" classname="Vtiger.GenericUtilities.BaseClass">
    <skipped/>
  </testcase> <!-- @AfterMethod amConfig -->
  <testcase name="@AfterClass acConfig" time="-1.690903609922E9" classname="Vtiger.GenericUtilities.BaseClass">
    <skipped/>
  </testcase> <!-- @AfterClass acConfig -->
  <testcase name="createContactWithOrgTest" time="0.0" classname="Vtiger.Contacts.test.CreateContactWithOrganizationTest">
    <skipped/>
  </testcase> <!-- createContactWithOrgTest -->
</testsuite> <!-- Default test -->
